src/middlewares/authMiddleware.js

```diff
@@ -0,0 +1,24 @@
+import { jwtService } from '../service/jwt.service.js';
+
+export const authMiddleware = (req, res, next) => {
+  const authorization = req.headers['authorization'] || '';
+  const [, token] = authorization.split(' ');
```

Comment:
Ensure that the authorization header follows the 'Bearer <token>' format. Consider adding a check to validate this format before splitting the string.